### DISCUSSION TITLE
flux:column does not include center alignment option

### DISCUSSION THREAD

**Discussion by jtesta66** [livewire/flux#684] [Feature Requests]
Using this code:
```
<flux:columns>
     <flux:column>Diner</flux:column>
</flux:columns>
```

It seems there is no way to automatically center the content within the generated <th>.  I see in the flux provided column.blade.php that there is an "align" option, but it doesn't accommodate center.

There is probably a way to force the content within the cell to be centered, but trying classes like "text-center" didn't seem to work for me.  It would be nice for this component to have options for "left", "right", and "center".

**Comment by legecha** [comment]
I agree. `<flux:column>` seems to translate to:

```
<th ...>
  <div class="flex ...">...</div>
</th>
```

It looks like adding `justify-center` onto the `div` manually in inspector achieves what we want, however I'm not css/tailwind or indeed fluxui savvy enough to know if it's possible without a new feature or by using customisations?

**Edit:** whether this is "correct" or not I'm not sure - but adding `flex justify-center` to the column works:

```
        <flux:columns>
            <flux:column class="flex justify-center">#</flux:column>
        </flux:columns>
```

**Reply by Kh4os-afk** [reply]
If you do

```
<flux:columns>
     <flux:column class="flex justify-center">1</flux:column>
     <flux:column class="flex justify-center">2</flux:column>
     <flux:column class="flex justify-center">3</flux:column>
</flux:columns>
```

All columns will be on top of each other 
That's not cool...
